src/test/java/racingcar/domain/car/ParticipantsTest.java

```diff
@@ -0,0 +1,96 @@
+package racingcar.domain.car;
+
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.Test;
+import racingcar.domain.random.MoveGen;
+import racingcar.domain.random.NoMoveGen;
+
+class ParticipantsTest {
+
+    @Test
+    public void 참가자_객체를_생성한다() {
+        //given
+        String[] names = {"jason", "pobi", "sung"};
+
+        //when
+        Participants participants = Participants.createCars(names);
+        List<Car> cars = participants.getParticipants();
+
+        //then
+        assertThat(cars.stream()
+            .map(car -> car.getName())
+            .collect(Collectors.toList())).containsExactlyInAnyOrder("jason", "pobi", "sung");
+    }
+
+    @Test
+    public void 참가_차량_생성_테스트() {
+        //given
+        String[] names = {"jason", "pobi"};
+
+        //when
+        Participants participants = Participants.createCars(names);
+        List<String> carNames = participants.getParticipants().stream()
+            .map(Car::getName)
+            .collect(Collectors.toList());
+
+        //then
+        assertThat(carNames).containsExactly("jason", "pobi");
+    }
+
+    @Test
+    public void race_이동_후에는_객체의_위치가_변해야한다() {
+        //given
+        List<Car> cars = Arrays.asList(Car.of("jason", 2), Car.of("pobi", 3));
+        Participants participants = new Participants(cars);
+
+        //when
+        Participants newParticipants = participants.race(new MoveGen());
```

Comment:
`MoveGen`과 `NoMoveGen` 객체가 없이 람다식을 통해서도 동일한 테스트 코드를 작성해볼 수 있을 것 같아요. 😃 
```suggestion
        Participants newParticipants = participants.race(() -> 5);
```

[함수형 인터페이스와 람다를 이용한 코드 중복 제거](https://tecoble.techcourse.co.kr/post/2021-11-21-functional-interface-with-lambda/) 글을 읽어보고 별도의 구현체 없이도 동일한 테스트를 할 수 있도록 변경해보세요. 🤗